src/integrationTest/java/uk/gov/hmcts/reform/hmc/controllers/HearingManagementControllerIT.java

```diff
@@ -1087,6 +1137,46 @@ void shouldReturn201_WhenHearingRequestHasValidData() throws Exception {
             .andReturn();
     }
 
+    @Test
+    @Sql(DELETE_HEARING_DATA_SCRIPT)
```

Comment:
Do we need this test method, as it looks as if this has been tested in `shouldReturn201_WhenHearingRequestHasPartyDetailsWhereOrgIdNull` on line 319 ?

Reply:
removed

Reply:
updated in 0ae612a810d04011ea9c0e28759e449f3a4fa43d